modules/tftp/tftp_api.rb

```diff
@@ -34,6 +34,11 @@ def create_default(variant)
       end
     end
 
+    post "/fetch_boot_image" do
+      log_halt(400, "TFTP: Wrong input parameters given. ") unless [params[:path], params[:url], params[:files]].all?
+      log_halt(500, "TFTP: Failed to fetch boot file: ") { Proxy::TFTP.fetch_boot_image(params[:path], params[:url], params[:files]) }
```

Comment:
I'd use a block for readability.
```suggestion
      log_halt(500, "TFTP: Failed to fetch boot file: ") do
        Proxy::TFTP.fetch_boot_image(params[:path], params[:url], params[:files])
      end
```

Reply:
Done.